src/diffusion.jl

```diff
@@ -96,13 +97,13 @@ The variables for diffusion problem:
 $(FIELDS)
 """
 struct Vars{Aphys, Atrans} <: AbstractVars
-    "tracer concentration"
+    "tracer concentration ``c``"
     c :: Aphys
-    "tracer concentration derivative ∂ₓc"
+    "tracer concentration derivative ``∂ₓc``"
    cx :: Aphys
-    "Fourier transform of tracer concentration"
+    "Fourier transform of tracer concentration ``c``"
    ch :: Atrans
-    "Fourier transform of tracer concentration derivative ∂ₓc"
+    "Fourier transform of tracer concentration derivative ``∂ₓc``"
```

Comment:
$\partial_{x}\hat{c}$?

Reply:
Hm. It’s actually the hat of the partial derivative. Let me see if it can be rendered in latex